projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -386,27 +385,43 @@
 
 .contact-form .contact-form__select-wrapper {
 	position: relative;
+
+	/*
+	 * Remove padding set on .grunion-field. We want the nested input to have no space around it.
+	 * Instead the padding is added to the `select` element itself.
+	 */
+	 padding: 0;
+}
+
+.contact-form .contact-form__select-wrapper .jetpack-field-dropdown__icon {
+	position: relative;
 }
 
-.contact-form .contact-form__select-wrapper::after {
+.contact-form .contact-form__select-wrapper .jetpack-field-dropdown__icon::after {
 	position: absolute;
 	inset-inline-end: 20px;
 	top: 50%;
-	color: var(--jetpack--contact-form--text-color);
 	content: "";
 	display: block;
-	width: 8px;
-	height: 8px;
-	z-index: 1;
-	border-bottom: 2px solid;
-	border-right: 2px solid;
-	transform: translateY(-50%) rotate(45deg);
+	width: calc(1em / 2.5);
+	height: calc(1em / 2.5);
+	border-bottom: calc(1em / 10) solid;
+	border-right: calc(1em / 10) solid;
+	color: inherit;
+	font-size: inherit;
+
+	/*
+		* The translateY is to center the arrow vertically.
+		* It's set to -75% to account for the fact that only the bottom-half of the element is visible.
+		* If we set it to -50%, the element would be centered, but not the visible part of the arrow.
+		*/
```

Comment:
A super tiny nit (feel free to ignore, it's just whitespace), but the indentation here is a bit weird.